webservices/sorting.py

```diff
@@ -77,8 +77,8 @@ def sort(query, key, model, aliases=None, join_columns=None, clear=False,
     # In this case, use the string name of the sort key.
     sort_model = (
         model
-        if len(query._entities) == 1 and hasattr(query._entities[0], 'mapper')
-        else None
+        # if len(query._entities) == 1 and hasattr(query._entities[0], 'mapper')
+        # else None
```

Comment:
I still need to figure out a solution for this. Select does not contain the attribute _entities.

Reply:
I think we have to use column_descriptions.